_topic_maps/_topic_map.yml

```diff
@@ -2592,6 +2592,29 @@ Topics:
 - Name: OpenShift sandboxed containers documentation has been moved
   File: sandboxed-containers-moved
 ---
+Name: Observability
+Dir: observability
+Distros: openshift-enterprise,openshift-origin
+Topics:
+- Name: Observability overview
+  File: index
+- Name: Power Monitoring
```

Comment:
Not fully sure the best way to frame this thought as it's kinda all over the place (and honestly might be more of a CS/PM-level question), but I was noticing the init cap "Power Monitoring" here in the new subdir, and then the existing `common-attributes` here, particularly the `:PM-operator:` one:
```
//power monitoring
:PM-title-c: Power monitoring for Red Hat OpenShift
:PM-title: power monitoring for Red Hat OpenShift
:PM-shortname: power monitoring
:PM-shortname-c: Power monitoring
:PM-operator: Power monitoring Operator
```
I assume the Operator capitalization has already gone through discussion/finalization, but the uppercase/lowercase/uppercase there struck me as fairly awkward mid-sentence. Not sure if potentially changing `:PM-operator:` to "Power Monitoring Operator" is in the cards now that the docs subdir is "Power Monitoring", or if the `_topic_map.yml` should be changed to `Name: Power monitoring` to better match the existing attributes, but just wanted to mention the discrepancy in some way.

Reply:
Hello @adellape and @linfraze This was a discussion we had with peer reviewer and engineering team when I worked on the docs. Basically, there is an exception for the Operator capitalization and we named the Operator based on what it is actually called in the console, so it should stay the way it is right now, thank you! :+1: 

Here for the reference: https://github.com/openshift/openshift-docs/pull/70711#discussion_r1466715734

Reply:
Also, I did not notice during my peer review, but the new subdir definitely needs to be changed to: Power monitoring instead of Power Monitoring, because that is the correct name. @linfraze I will make that change in the new PR as we discussed.